drizzlepac/haputils/processing_utils.py

```diff
@@ -169,6 +169,11 @@ def refine_product_headers(product, total_obj_list):
     # Re-format ACS filter specification
     if phdu['instrume'] == 'ACS':
         phdu['filter'] = get_acs_filters(hdu, delimiter=';')
+        
+    # WFPC2 update aperture if in poller file
```

Comment:
Again, to avoid confusion for a future maintainer, is would be better to name the product attribute something like self.scale_aperture and have a comment which indicates how it is different than the original RAW APERTURE keyword.